code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -97,6 +97,71 @@
 	name = "trophy rack"
 	desc = "It's useful for both carrying extra gear and proudly declaring your insanity."
 	icon_state = "cultpack"
+	var/obj/item/organ/external/head/LS
+	var/obj/item/organ/external/head/MS
+	var/obj/item/organ/external/head/RS
+	var/mutable_appearance/LSma
+	var/mutable_appearance/MSma
+	var/mutable_appearance/RSma
+	var/list/placement
+
+/obj/item/storage/backpack/cultpack/attackby(obj/item/I, mob/living/user)
+	..()
+	if(istype(I, /obj/item/organ/external/head))
+		placement = input(user, "Where would you like to put the head?") in list(!LS ? "Left Spike" : "", !MS ? "Middle Spike" : "", !RS ? "Right Spike" : "", "Bag")
+		switch(placement)
+			if("Left Spike")
+				LS = I
+				user.visible_message("[user] sticks [I] onto the left spike of the trophy rack","You stick [I] on the left spike of the trophy rack.")
+				LSma = mutable_appearance(I.icon, I.icon_state)
+				var/matrix/M = matrix()
+				M.Turn(335)
+				M.Translate(-4,0)
+				I.transform = M
+				LSma.transform = M
+				add_overlay(LSma)
```

Comment:
Can extract this part and `playsound` from all cases to reduce code duping

Reply:
Fixed